### PR TITLE
Realtime Multilingual Chat App

### PR DESCRIPTION
**Pull request by sarahchima**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    //Subscribe the user to the messages of the channel. So the use rwill receive each message sent to the test channel.
+
+    channel.subscribe("text", function(message) {
+        const selectedLanguage = $("#languageSelector").find(":selected").val();
+        translateText(message, selectedLanguage)
+    });
+
+    //This gets the data of other users as they publish to the channel.
+    channel.subscribe("user", (data) => {
+        var dataObj = JSON.parse(JSON.stringify(data));
```
Comment (Srushtika) [inline index.js]: The data object is already in JSON format by default. Can you update this in a new commit please? No need to do a new step-wise commit.

Comment (Srushtika) [reply index.js]: Please also update the tutorial one as well after having tested it, thanks.

**Review by Srushtika — APPROVED** [review]
One small comment otherwise looks good thanks!

re the file `public/bundle.js` - what does it do? I can see the contents of that file? Can it be removed?

**Comment by Srushtika** [comment]
@sarahchima please also update the README file to contain details of this specific project/ tutorial. Thanks!

**Comment by sarahchima** [comment]
> One small comment otherwise looks good thanks!
> 
> re the file `public/bundle.js` - what does it do? I can see the contents of that file? Can it be removed?

The client-side javascript is written in ES6 and has to be compiled first. The bundle.js file is where is compiled code is and that's what serves the frontend.

**Review by Srushtika — APPROVED** [review]

**Pull request merged**